### ISSUE TITLE
PTN000897489 (PTHR11523)

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#3194]
PTN000897489 adds "contributes_to sodium:potassium-exchanging ATPase activity (GO:0005391)" to members of the SODIUM/POTASSIUM-DEPENDENT ATPASE BETA SUBUNIT (PTHR11523) family.

But these members all encode 'beta subunits' of the sodium:potassium-exchanging ATPase complex complex, which are non-catalytic (see description of GO:0005890).

Can you block this propagation please?

The source annotations ought to be checked/fixed too:

(DROME|FlyBase=FBgn0015776|UniProtKB=Q24046) - I've fixed this
(DROME|FlyBase=FBgn0015777|UniProtKB=Q24048) - I've fixed this

(MOUSE|MGI=MGI=88108|UniProtKB=P14094) from BHFL
(HUMAN|HGNC=804|UniProtKB=P05026) from BHFL & ARUK
(HUMAN|HGNC=805|UniProtKB=P14415) from BHFL
(HUMAN|HGNC=806|UniProtKB=P54709) from BHFL & ARUK
(PIG|Gene=ATP1B1|UniProtKB=P05027) from BHFL & ARUK
Can @RLovering look at these?

@hattrill

**Comment by RLovering** [comment]
Hi

I guess this is why I didn't like contributes to being removed

It is going to take me a while to do these as there are also other annotations in these records to fix if this is only regulating the activity rather than being part of the activity ie it regulates ion transport rather than being involved in the transport. I have done a couple but still got a few to do

(MOUSE|MGI=MGI=88108|UniProtKB=P14094) from BHFL - fixed
(HUMAN|HGNC=804|UniProtKB=P05026) from BHFL & ARUK - fixed
(HUMAN|HGNC=805|UniProtKB=P14415) from BHFL -  to do
(HUMAN|HGNC=806|UniProtKB=P54709) from BHFL & ARUK -  to do
(PIG|Gene=ATP1B1|UniProtKB=P05027) from BHFL & ARUK -  to do

Best

Ruth

**Comment by sjm41** [comment]
Hi Ruth,
Probably good idea to review, though I don't think the BP annotations will necessarily need to be changed ('involved in' -> 'regulates') in these cases - I think beta subunits can still be 'involved in' transport, even though their MF is non-catalytic within the complex.  Thoughts @hattrill?

**Comment by RLovering** [comment]
Hi Steven

I would have thought that if a protein regulates a transporter then it is involved in regulating transport rather than being involved in transport. @vanaukenk do you have any thoughts on this?

Thanks
Ruth

**Comment by sjm41** [comment]
Hi @pgaudet - I just ran into this issue again.  Can you remove "contributes_to sodium:potassium-exchanging ATPase activity (GO:0005391)" from this node please? Thanks.

**Comment by RLovering** [comment]
Hi Kimberly

please could you confirm whether a beta subunit of a sodium:potassium-exchanging ATPase activity is regulating the transport or participating in the transport. 

As the transporter does not transport without the beta subunit I would rather it was annotated as contributing to the activity and participating in transport. 

It just seems counter intuitive to say something regulates the transporter but is involved in transport, and more logical to say if something regulates a transporter that it regulates transport

Thanks

Ruth

**Comment by vanaukenk** [comment]
@RLovering 

I think it would help to understand, if possible, more about the mechanism of action of the beta subunit. 

@sjm Do you know more about how the beta subunits function?

**Comment by RLovering** [comment]
Thanks Kimberly
https://www.nature.com/articles/srep27738 provides a 3D picture of the alpha and beta complex and states: According to our model, binding Aβ(1-42) hinders the movement of the alpha- and beta-subunit towards each other during the catalytic cycle23, disrupting the Na,K-ATPase function.

https://pubmed.ncbi.nlm.nih.gov/10636900/ provides a lot of experimental evidence (human Na,K-ATPase isozymes in Xenopus oocytes): Transport and pharmacological properties of nine different human Na, K-ATPase isozymes. Note that all experiments include alpha with beta subunits.

This article is interesting https://link.springer.com/article/10.1007/s12035-014-9076-z with the following statement: P2C-type ATPases are conformed by α and β subunits, and Na+/K+-ATPase has a third subunit called γ or FXYD. α subunit contains the catalytic site, and in Na+/K+-ATPase and one type of H+/K+-ATPase, is the site of ouabain binding...... β subunit is a minor subunit that contributes to the K+ affinity in its binding site and may contribute to the trafficking and delivery of Na+/K+-ATPase to the cell membrane [26]. The β subunit also contributes to the function of the α subunit. The third subunit of Na+/K+-ATPase, named FXYD, is also a regulatory unit and contributes to the affinity of Na+ to its binding site. Fig 1 legends states: Na+/K+-ATPase is composed by three subunits, α, β, and FXYD. Only α subunit is the catalytic one and is the place where ion exchange occurs....β and FXYD subunits are to stabilize the ion binding and the activity of the enzyme.

Probably Birgit would be better than me at interpreting the information in these 3 papers. But I guess my feeling now is that this transporter includes multiple functions, ie binding to the substrate as well as binding to the activator/inhibitor to regulate the activity, plus the use of energy to facilitate the transport against a gradient. So these papers reinforce my view that the beta and alpha subunits both contribute to the 'transporter' activity and are both involved in transport. I am sitting on the fence wrt to the FXYD (gamma) subunit as the oocyte expts do not include FXYD(although I guess it is possible that the oocyte expresses the xenopus endogenous FXYD ortholog.

Hope this helps

Best

Ruth

**Comment by sjm41** [comment]
Thanks Ruth

I think part of the problem here is that term in question (sodium:potassium-exchanging ATPase activity (GO:0005391)) is one of those combinatorial terms that has both a transport and ATPase reaction components, which aren't really separable.

Nonetheless, I'd focus on these statements from the refs you found:
- As you said, https://link.springer.com/article/10.1007/s12035-014-9076-z  says "Only α subunit is the catalytic one and is the place where ion exchange occurs....β and FXYD subunits are to stabilize the ion binding and the activity of the enzyme."

- And the intro of https://pubmed.ncbi.nlm.nih.gov/10636900/ says "The minimal functional unit of Na,K-ATPase is composed of an α and β subunit. The α subunit has 10 membrane-spanning domains and exposes the N and C terminus to the cytoplasmic side while the β subunit is a type II glycoprotein with a single transmembrane segment, a short cytoplasmic tail, and a large ectodomain. **The α subunit carries the functional properties of the Na,K-ATPase, namely it binds and transports the cations, hydrolyzes ATP, and is intermediately phosphorylated.** Furthermore, the α subunit bears the binding site for cardiac glycosides (for review, see Ref. 2). The β subunit is necessary for the structural and functional maturation of the α subunit and also influences the K+ and Na+activation kinetics of mature pumps (for references, see Ref. 3).

From these, it seems clear to me that only the alpha subunit should be annotated with "sodium:potassium-exchanging ATPase activity". The beta subunit can have "ATPase activator activity (GO:0001671)" (in terms of MF).

Thoughts from @hattrill and @bmeldal on this one?

FYI, looking at the PAINT node and P2GO just now, the remaining 'problematic' annotations are to human ATP1B2 and ATP1B3 :
(HUMAN|HGNC=805|UniProtKB=P14415) - 2 annotations by BHFL from PMID:10636900, PMID:19542013
(HUMAN|HGNC=806|UniProtKB=P54709) - 3 annotations by ARUK/BHFL from PMIDs 16861705, 10636900,19542013

**Comment by hattrill** [comment]
In this case, the beta SU is definitely regulatory/folding/cellular loc and does not contribute to the core function transporter function. Regulator would be the most accurate description of function - a bit like cyclins are part of a CDK complex and regulate kinase function.

**Comment by RLovering** [comment]
Hi Helen

OK will go with this decision. However, this still leaves open the question of whether the beta subunits should be associated with regulating transport or involved in transport.

I will update all records I can find once this BP decision is made. 

Thanks

Ruth

**Comment by hattrill** [comment]
involved in transport, would be my advice - whereas different complex SUs can have different MFs, they are all part of the same process.

**Comment by bmeldal** [comment]
Hi all,

I agree with @hattrill . 

Beta SU looks like the regulator or activator (you decide that one!):
child of `GO:0030234 enzyme regulator activity` (would be a sibling of your beloved GO:0140110 transcription regulator activity ;-) ), maybe `GO:0060590 ATPase regulator activity` (which has activator child term if required)? You can always request the more specific child of `sodium:potassium-exchanging ATPase regulator activity`.

I also agree that it's *involve_in* `transport` as the transport relies directly on beta SU binding.

Even FXYD can probably be annotate the same as beta SU, at least with *regulator*, maybe not *activator* child.

Birgit

**Comment by RLovering** [comment]
Ok 

all great advice. I will request that the definition for the transport BP term is modified:
GO:0006810 transport
Definition
The directed movement of substances (such as macromolecules, small molecules, ions) or cellular components (such as complexes and organelles) into, out of or within a cell, or between cells, or within a multicellular organism by means of some agent such as a transporter, pore or motor protein. 

Change to:
Definition
The directed movement of substances (such as macromolecules, small molecules, ions) or cellular components (such as complexes and organelles) into, out of or within a cell, or between cells, or within a multicellular organism by means of some agent such as a pore, motor protein or transporter protein or complex. 

There are 350 annotations associated with 250 proteins using a transporter activity term and a contributes to qualifier are you going to set up a annotation review for this along with recommendations for curation of the BP transport for protein in the transporter complex rather than regulation of transport. UCL has over 60 proteins to review

Thanks

Ruth

Thanks

Ruth

**Comment by sjm41** [comment]
_There are 350 annotations associated with 250 proteins using a transporter activity term and a contributes to qualifier are you going to set up a annotation review_

This idea (need?) has been mentioned several times by several people (not just for transporter activity, but all contributes_to annotations involving a subunit of a complex). I'm just disputing/fixing them in fly annotations (or non-fly annotations that affect fly annotations via PAINT etc) when I find them.  A systematic review and fix would be great - but won't be a small task, and I don't know how to go about organising it myself....

**Comment by bmeldal** [comment]
> _There are 350 annotations associated with 250 proteins using a transporter activity term and a contributes to qualifier are you going to set up a annotation review_
> 
> This idea (need?) has been mentioned several times by several people (not just for transporter activity, but all contributes_to annotations involving a subunit of a complex). I'm just disputing/fixing them in fly annotations (or non-fly annotations that affect fly annotations via PAINT etc) when I find them. A systematic review and fix would be great - but won't be a small task, and I don't know how to go about organising it myself....

This was the final action point for the complex WG after the May 2020 GOC mtg. @pgaudet normally pulls out and creates lists for annotation reviews but in this case many new terms for regulators may need to me created. One idea we muted was to have an annotation jamboree like the one "we" (I wasn't involved!) did for transcription. I think it's in @pgaudet and @vanaukenk court to make a decision on this. Sorry for passing the buck but I'm not a GO PI (I'm not a PI, full stop) ;-)

**Comment by hattrill** [comment]
Most transporter annotations should be ok with contributes_to, as they are genuine heteromers with SUs which have roles in transport. Regulatory subunits are less frequently seen. I think that this particular term branch is tricky - would involve quite a lot of cross-ref'ing.
I don't think that we came up with a fantastic idea in the last meeting on how to do the review - we need to think again, otherwise it involves a lot of xref'ing. Perhaps, one way is to take the PAINT annotations that have propagated contributes_to (as, even if there is just one with the qualifier, it will add it) and then we could do a protein-class based review, which would be easier.

**Comment by sjm41** [comment]
@hattrill Running a quick (and self-centred!) query on FlyBase for genes GO-annotated as part of a protein complex AND having 'contributes_to' a catalytic activity gives me a list of 200 unique Dmel genes (320 annotations) that we could review.  As you say, these could be further divided/prioritised based on source (e.g. PAINT) or protein class (e.g. transporters).

**Comment by hattrill** [comment]
@sjm41 remind me next week - a did a bit of an analysis wrt terms associated with contributes_to and had some ideas about what strategy could be used. For us, because of gene groups, things might be a bit simplier that for other groups....but if we did some sort of cross-ref with PANTHER families.....

**Comment by sjm41** [comment]
@hattrill Sounds good, will do. I've probably already reviewed many of the subunits with contributes_to catalytic activity, but will surely have missed some (and some will have been added more recently).  A FlyBase-focussed review (if its relatively simpler) of this issue would of course identify many of the PAINT-derived contributes_to issues, so would have wider benefit. :-)

**Comment by ValWood** [comment]
Hang on. In principle, this all sounds great but let's generalise this.

Above we are saying if a GP  activates another GP subunit we annotate to 

Gene1 _enables_ **blah_activator** has_input Gene2 _enables_ **blah**
part_of _process_

not
Gene1 _enables_ **blah_activator** has_input Gene2 _enables_ **blah**
part_of _positive regulation of process_

It, therefore, follows that:

Gene1 _enables_ **blah_inhibitor** has_input Gene2 _enables_ **blah**
part_of _process_

not to
Gene1 _enables_ **blah_activator** has_input Gene2 _enables_ **blah**
part_of  **negative regulation of process**

Is that true? (I think it has to be because we need to be consistent, we can't change this on. a case by case basis).

So for example, this annotation:

cut2 securin, sister chromatid separation inhibitor

Cut2  _enables_ **endopeptidase inhibitor activity** has_input cut1 _enables**
**endopeptidase activity**  part_of **negative regulation of mitotic sister chromatid separation**

is incorrect and it should really be

Cut2  _enables_ **endopeptidase inhibitor activity** has_input cut1 _enables**
**endopeptidase activity**  part_of **mitotic sister chromatid separation**

This kind of makes sense, because the control is via some upstream signalling  to degrade the inhibitor of the activator (the peptidase) to release cohesin. The endopeptidase is an inhibitor, and the petidase is an activator, but they are not really exerting the control. They are only implementing it.

If this is the case, what we are saying here, is that within process regulation of activities should not be captured as 'regulation of process'.   I think this is what GO-CAM tries to do, but it has never been communicated as a 'rule' and I suspect people still use "regulation of process" terms when curating in Noctua.

I am very  happy to curate like this. This is part of what I've been trying to explain that we inconsistent about within process, and upstream regulation. It feels correct,  but sometimes without a rule in place, it seems more natural to use the "regulation of" process term.

Also, there are plenty of cases where the within-process regulation is hard-wired into the ontology. Here for instance:

actin nucleation (GO:0045010)
which is clearly  part of cytoskeleton organization
has a parent
positive regulation of actin filament polymerization
which makes it 
positive regulation of cytoskeleton organization (GO:0051495)

Can this  issue  be addressed at the Noctua workshop because everything we curate hinges on the answer to this?


@thomaspd 
@cmungall 
@pgaudet 


Also please can we get the alerting back. I only saw this ticket  because I spotted @RLovering  corresponding ticket on the ontology tracker
https://github.com/geneontology/go-ontology/issues/20292

**Comment by ValWood** [comment]
same here
GO:0000752    agglutination involved in conjugation with cellular fusion
has regulation "positive regulation of conjugation with cellular fusion" in it's ancestry
via GO:0000749   response to pheromone triggering conjugation with cellular fusion
phew,

GO:0000752    agglutination involved in conjugation with cellular fusion
needs to be obsoleted anyways

GO:0098631    cell adhesion mediator activity
part of   conjugation with cellular fusion
would probably do here.

**Comment by hattrill** [comment]
I am not sure if we can always make the generalisation - of always pairing activator/inhibitor of MF to involved_in/part_of BP rather than involved_in/part_of regulation of BP.
The question is - is this MF required for the completion of the BP?

So, X is an inhibitor of an MF required for a BP, then it is negatively regulating that BP.
If X is an activator of an MF, then is this an obligatory component of the BP or just altering the flux of the BP in favour of the end point? In the former case, it is involved_in/part_of and the latter, positively regulating BP.

As for what constitutes "upstream" - I think we are well over-due a discussion about what "upstream" means.

Yes - I agree, that some of the hardwired inferences in the GO, are inconsistent and unhelpful.

**Comment by ValWood** [comment]
> The question is - is this MF required for the completion of the BP?

So is it essential for the process? I don't think that can be the criteria. We have lots of gene products that are involved in a process, but not essential for that process.

For example in yeast the alp14 is involved in chromosome segregation (many aspects) but isn't essential for chromosome segregation, because there are often many back up pathways and mechanisms to ensure fidelity. 

This doesn't mean that these activities are regulating a process just because they aren't absolutely required? That criteria would really restrict what could be annotated as involved_in a process.

**Comment by hattrill** [comment]
You misunderstand. Shouldn't base this on just the KO outcome but the required pathway sequence of molecular events - A must happen before B, etc. What steps are required for the execution and what gps perform these.

If an MF can be performed by different enitities, but that step needs to take place, then it is part_of (so if you have mulitple kinase isoforms and they are redundant, then all should be part_of)

If a complex is involved, then all complex components should be part_of as they are all part of a functional unit. (But, if you based this on any sort of KO/depletion exp, then you will always see that some complex members are more crucial than others.)

**Comment by ValWood** [comment]
So I think what I posited in the first comment still holds. In these cases the gene products I mentioned are definitely part of "chromosome segregation" . We could pose the question another  (simpler) way. Can a gene product be (simultaneously) both part of, and regulate the same process.

**Comment by ValWood** [comment]
> If an MF can be performed by different enitities, but that step needs to take place, then it is part_of

I don't think this is sufficient either.

A signalling pathway regulating a process is necessary, it still needs to take place in the ordered sequence of events.
That doesn't make it part of the process?

**Comment by hattrill** [comment]
"Can a gene product be (simultaneously) both part of, and regulate the same process."
Yes - there are a number of pathways in which components in the "off state" act as negative regulators and in the "on" state are part of the pathway process - kinases are particularly a feature in this. So, I annotate the suppression of the pathway as "negative regulation of XXX" and the where it acts as a step in the process as involved in "XXX". Hedgehog signaling has many instances of this.

**Comment by hattrill** [comment]
"A signalling pathway regulating a process is necessary, it still needs to take place in the ordered sequence of events." 

If it is absolutely necessary e.g. MAP kinase cascade - each gp regulates the next, but they are part of the MAP kinase cascade - the scaffold is also absolutely necessary, too. But there are other kinases which may impinge on the pathway under certain conditions and increase the activity of a kinase in the pathway (therefore regulating).

**Comment by ValWood** [comment]
So for signalling pathways sure, because the purpose is regulation, and the effect of the signalling components is different in the activated or inactivated state.

I think from the transcription  work it turned out that the gene products which are part_of transcription  were then not also regulating transcription (I might be wrong here, but this is what I remember). So I was wondering 
a) how general this is?
and
b) what are the exceptions?

I did this exercise before a few years back. I intend to run through it again in the next few weeks to see what changed and document the different ways we use regulation (regulation of function, regulation of process, regulation of pathway choice, signalling etc).

**Comment by hattrill** [comment]
And, what I am struggling with is: how does the concept of regulation work with the new gp2term rels?

**Comment by ValWood** [comment]
That isn't clear to me either...

**Comment by sjm41** [comment]
Hi,
This ticket was really just about a MF term on a specific PAINT node....
Can the wider (important!) points being raised/discussed here go into a new ticket, and/or be put on the agenda for a meeting?

**Comment by hattrill** [comment]
Sorry, Steven, but I think by weight of comments, it's mine and Val's ticket now ;-)

**Comment by sjm41** [comment]
Hmmm.  That makes it hard to close this ticket....
Suggest you rename the ticket if you want to keep discussing the BP regulates issue here...

**Comment by hattrill** [comment]
Go ahead and close it. We will dicuss this elsewhere :-)

**Comment by sjm41** [comment]
Wrt to original topic and closing the ticket, these annotations to "sodium:potassium-exchanging ATPase activity(CONTRIBUTES_TO)"still need to be fixed and/or @pgaudet could change the annotations on the paint node directly:
(HUMAN|HGNC=805|UniProtKB=P14415) - 2 annotations by BHFL from PMID:10636900, PMID:19542013
(HUMAN|HGNC=806|UniProtKB=P54709) - 3 annotations by ARUK/BHFL from PMIDs 16861705, 10636900,19542013

**Comment by RLovering** [comment]
Hi All

https://www.jbc.org/content/275/3/1976.long states: The β subunit is necessary for the structural and functional maturation of the α subunit and also influences the K+ and Na+activation kinetics of mature pumps

So I think that these can be associated with both 'transport' and 'regulation of transport' terms. In addition to the 'activator' term I have added the protein-macromolecule adaptor activity.

I have decided that I am not comfortable decribing the subunits as only activators if they are essential subunits for a functional transporter. 

All of the requested annotations have been changed.

I have created a ticket for the review of 460 contributes to UCL annotations. Not sure when these will be done
https://github.com/geneontology/go-annotation/issues/3510

I will leave you to close the ticket once you have confirmed the PAINT annotations are in order

Best

Ruth

**Comment by sjm41** [comment]
Thanks Ruth! I'll close this now given the source annotations (feeding PTN000897489) have been corrected and will update in PAINT in time. @pgaudet - please reopen if you wish to make direct edits to the PAINT node sooner.

**Comment by ValWood** [comment]
Hi Ruth,

My  current understanding is that just because the regulatory subunit is regulating the activity of the transporter, it is not necessarily  regulating the process (because it is part of the process).

So in this case the annotation could be 
transporter activator part_of transport
(because, although this gene product is required for the activity of the transporter, it is not  the control point for the process).

I think this fits with the GO-Cam paradigm, but I haven't seen this documented anywhere for either  GO-cam or traditional annotation, so I'm not sure that it is correct. It does  align with the transcription work though, where, if a general transcription initiation factors  is a  regulator of a specific activity (GTF kinases for example), they are annotated directly to "transcription initiation" and not to "regulation of transciption initiation".

This is why I asked yesterday if anyone had examples  (other than signalling)  where something was both.... part_of and regulating a process.]


Val

**Comment by pgaudet** [comment]
I'll try to edit the tree.

**Comment by bmeldal** [comment]
> Hi Ruth,
> 
> My current understanding is that just because the regulatory subunit is regulating the activity of the transporter, it is not necessarily regulating the process (because it is part of the process).
> 
> **So in this case the annotation could be
> transporter activator part_of transport
> (because, although this gene product is required for the activity of the transporter, it is not the control point for the process).**
> 
> I think this fits with the GO-Cam paradigm, but I haven't seen this documented anywhere for either GO-cam or traditional annotation, so I'm not sure that it is correct. It does align with the transcription work though, where, if a general transcription initiation factors is a regulator of a specific activity (GTF kinases for example), they are annotated directly to "transcription initiation" and not to "regulation of transciption initiation".
> 
> This is why I asked yesterday if anyone had examples (other than signalling) where something was both.... part_of and regulating a process.]
> 
> Val

Just caught up with this story ;-)

I agree with Val, if a subunit *regulates* the activity as part of a complex (the beta subunit in the above example) then it's directly *involved_in* the process that this complex is *part_of/involved_in* (those ROs are getting confusing!).

I also agree with an earlier comment about needing to clearly define *part_of/involved_in* vs *regulates* a process. I seem to remember that there have been discussions about clearly defining start and end points of processes and that would be crucial. But it's also kind of shoehorning a complex biological system into discrete packages which isn't really possible (throws hands in the air...). E.g. a catalyst acts tangentially to the process, may actually be absolutely required for it but is not part of the cascade. Is it *involved_in* or *regulates*???

**Comment by pgaudet** [comment]
I think those questions are covered in many other places - I'll close this now that that the annotation is removed from the family. 

Thanks everyone for a great discussion !
